app/sections/single-product/index.tsx

```
@@ -189,11 +179,11 @@ const SingleProduct = forwardRef<HTMLElement, SingleProductProps>(
                 </p>
                 {children}
                 <p
-                  className="leading-relaxed fade-up line-clamp-5"
+                  className="fade-up line-clamp-5 leading-relaxed"
                   suppressHydrationWarning
                   dangerouslySetInnerHTML={{ __html: product?.summary }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)